twilio/twiml/fax_response.py

```diff
@@ -7,6 +7,7 @@
 """
 
 import json
+from twilio.base.obsolete import deprecated_method
```

Comment:
nit: Premature.

Reply:
I agree, and looked into the conditional generation of these imports, but found they're generated earlier in the control flow than when we know whether there are legacy functions to generate. I'll look into adding some logic to handle this better

Reply:
Fixed